aws-lc-rs/tests/data/rsa_from_pkcs8_tests.txt

```diff
@@ -22,67 +22,50 @@ Input = 30821242020100300d06092a864886f70d01010105000482122c30821228020100028204
 
 # RSA 512-bit key (too small)
 Input = 30820154020100300d06092a864886f70d01010105000482013e3082013a02010002410094f95f85cae61920ffca1deeb2135de7bf20fc05523198433178e74bd3090445f5089488a69bed2984425e497e255abe329129f5b166ad8b204768b8e374fbdd020301000102403e65280e088a513212105c731220ac6f190d3ea117e8efc60cdd7fbab15c106e7a9c8d9be4c0e03470742bc2d0e97233656b771be841a00e0957824c6091fc9d022100e7689ca4d01c8f0a190defcf4bd59038f01879605cba34d98a8e8cd04b00974f022100a4ce28ec8938a3012604174433260b8e0ed783fb2720c4c34abe004f7ea9ef13022100c94768d8c25bd4df07a3e3aeff5b27331f436a6fb2b814381418a1ca9b324c69022027f7efa8dde6a471a4dc9d1fe71fb4f1b0dbc7a0636ee9094bd9248e22f3e78d02201d849cc8c96c732c542a5ba82c7a7a474e05aeb5fb5689ee6a461e3ad545bd85
-Error = TooSmall
+Error = Unspecified
```

Comment:
Why are we losing visibility into why the tests are failing?

Reply:
This required duplicating logic AWS-LC validation logic checks on RSA key sizes, so in order to not repeat / duplicate checks we are opting to remove this level of specificity.  So the error is till `KeyReject` in aws-lc-rs, but the string message contain is now `Unspecified`.